koku/api/report/aws/openshift/query_handler.py

```diff
@@ -129,7 +129,11 @@ def __init__(self, parameters):
             parameters    (QueryParameters): parameter object for query
 
         """
-        self._mapper = OCPAWSProviderMap(provider=self.provider, report_type=parameters.report_type)
+        self._mapper = OCPAWSProviderMap(
+            provider=self.provider,
+            report_type=parameters.report_type,
+            cost_type=parameters.parameters.get("cost_type"),
```

Comment:
It probably makes sense to add a `cost_type` property to QueryParameters like we do [here](https://github.com/project-koku/koku/blob/main/koku/api/query_params.py#L463-L466), but not a super critical change.